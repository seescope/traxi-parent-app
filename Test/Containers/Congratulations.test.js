import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import CongratulationsComponent from '../../App/Containers/Congratulations';

const mockStore = configureStore();
const testStore = mockStore({
  kidsState: {
    'abc-123': {
      name: 'John Bobson',
    },
  },
  setupState: {
    kidUUID: 'abc-123',
  },
});

const Congratulations = () => (
  <Provider store={testStore}>
    <CongratulationsComponent />
  </Provider>
);

describe('<Congratulations />', () => {
  it('renders the Congratulations component', () => {
    const tree = renderer.create(<Congratulations />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
