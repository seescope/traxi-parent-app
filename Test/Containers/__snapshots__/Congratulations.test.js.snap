// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Congratulations /> renders the Congratulations component 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F0F2F7",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "paddingTop": 40,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "#545454",
          "fontFamily": "Raleway-ExtraBold",
          "fontSize": 26,
          "paddingHorizontal": 55,
          "textAlign": "center",
        }
      }
    >
      John
       is being monitored!
    </Text>
    <View
      style={
        Object {
          "height": 32,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
            "color": "#545454",
            "fontFamily": "Raleway-Regular",
            "fontSize": 16,
            "paddingHorizontal": 22,
            "textAlign": "left",
          },
          undefined,
        ]
      }
    >
      Congratulations,
       
      John
       
      is being monitored by Traxi. You will soon be able to see what they are doing on their device.
      
      
      
      While we collect data from
       
      John
      's device, there's just two more things we need to do.
    </Text>
    <View
      style={
        Object {
          "height": 32,
        }
      }
    />
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "flexDirection": "row",
          "justifyContent": "center",
          "marginTop": 16,
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#448AFF",
            "fontFamily": "Raleway-Regular",
            "fontSize": 20,
          }
        }
      >
        Next step
         
      </Text>
    </View>
  </View>
</View>
`;
